l10n_it_account_stamp/migrations/13.0.1.0.0/post-migrate.py

```diff
@@ -0,0 +1,19 @@
+#  Copyright 2022 Alex Comba - Agile Business Group
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        update account_move
+        set
+            tax_stamp = inv.tax_stamp
```

Comment:
se il campo non viene ricalcolato alla migrazione in assenza di questo codice direi che si tratta più di una `fix` (non solo una `add`).

Reply:
Grazie, ho modificato il titolo della PR.